tck/src/main/java/org/eclipse/microprofile/config/tck/ConfigProviderTest.java

```diff
@@ -155,13 +155,15 @@ public void testInjectedConfigSerializable() {
         ByteArrayOutputStream byteArrayOutputStream = new ByteArrayOutputStream();
         try (ObjectOutputStream out = new ObjectOutputStream(byteArrayOutputStream)) {
             out.writeObject(config);
-        } catch (IOException ex) {
```

Comment:
Are the formatting changes in this and `ConverterTest` required for JDK 11?

Reply:
I was thinking the same, but just assumed it was due to checkstyle.  But if not, then the formatting changes should be left out probably...

Reply:
+1

Reply:
It was to sastisfy the checkstyle.